app/cli/internal/policy/init/templates/example-policy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: policy.chainloop.dev/v1
```

Comment:
let's add some headers indicating that this is generated by the CLI, the link to the docs, etc, at the top

Other than that this is nice